pyproject.toml

```diff
@@ -1,12 +1,12 @@
 [tool.poetry]
 name = "promptolution"
-version = "1.0.1"
+version = "1.1.0"
 description = ""
 authors = ["Tom Zehle, Moritz Schlager, Timo Heiß"]
 readme = "README.md"
 
 [tool.poetry.dependencies]
-python = "^3.11"
+python = "^3.9"
```

Comment:
can we add this information to the release notes? "enable support for python versions from 3.9 onwards (previously 3.11)" or similar